.azure/OneBranch.Official.yml

```diff
@@ -204,7 +188,6 @@ extends:
       dependsOn:
       - build_windows
       - build_uwp
-      - build_windows_xdp
       jobs:
       - template: .azure/obtemplates/build-nuget.yml@self
```

Comment:
I think this yaml and associated scripts will need to be updated to not do anything XDP specific any more.